test/python/unittest/API/APITest.py

```diff
@@ -540,6 +545,11 @@ def testSliceRead(self):
         pos_y = electrons["position"]["y"]
         w = electrons["weighting"][api.Record_Component.SCALAR]
 
+        if found_numpy:
+            np.testing.assert_allclose(electrons["position"].unit_dimension,
```

Comment:
Does this work for momentum as well?

Reply:
yep, all openPMD records :)

I just tested it on a mesh and a particle record to see it works.